pippo-core/src/main/java/ro/pippo/core/route/UrlResourceHandler.java

```diff
@@ -49,6 +53,59 @@ public final void handleResource(String resourcePath, RouteContext routeContext)
 
     public abstract URL getResourceUrl(String resourcePath);
 
+    protected String getResourceVersion(String resourcePath) {
+        URL resourceUrl = getResourceUrl(resourcePath);
+        try {
+            return Long.toString(resourceUrl.openConnection().getLastModified());
+        } catch (IOException e) {
+            throw new PippoRuntimeException("Failed to read lastModified property for {}", e, resourceUrl);
```

Comment:
If we are trying to access a resource that does not exist I think we're going to throw a `500` for the request instead of a `404` because we are throwing this exception.

We might consider swallowing the `IOException` and returning `null` if we can't `getLastModified`.  It won't hurt anything, I don't think, and it will allow more understandable results (i.e. 404).